pkg/plugins/queries.go

```diff
@@ -64,7 +64,10 @@ func GetEnabledPlugins(orgId int64) (*EnabledPlugins, error) {
 	}
 
 	for pluginId, app := range Apps {
-		if b, ok := pluginSettingMap[pluginId]; ok {
+		if app.AutoEnabled {
```

Comment:
This won't actually enable apps though correct? Or at least, it won't set the plugin data - It will just pin it in the nav, so maybe `AutoPinned`/`AutoPinnedToNav` is a better name?

Reply:
it will once I figured out how, at least that is the plan, but was harder than I expected